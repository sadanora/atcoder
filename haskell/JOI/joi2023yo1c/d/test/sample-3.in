10
766 152 595 926 663 509 368 595 175 622
