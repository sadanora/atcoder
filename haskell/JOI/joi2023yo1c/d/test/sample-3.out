9
1
5
10
8
4
3
5
2
7
